index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <link rel="stylesheet" href="index.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Profile</title>
+</head>
+
+<body>
+    <header>
+        <nav id="nav">
+            <div class="nav">
+                <a class="nav__logo" href="#">My Logo</a>
+    
+                <img class="nav__hambuger" src="images/hambuger.svg" alt="Icon">
+    
+            </div>
+        </nav>
```

Comment:
- [ ] I think the logo is not a part of the navbar of a website so please remove the logo element from the `<nav>`.